### DISCUSSION TITLE
el-table-column 中 type="expand"时如何自定义expand icon

### DISCUSSION THREAD

**Discussion by liangdz09** [element-plus/element-plus#9443] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table-column`

## Description

el-table-column component supports custom expand icon when type="expand"

### Additional comments

>这个图标可以作为默认图标。应该支持自定义图标

**Comment by kekexunxun** [comment]
可以考虑为table-column增加这个属性，这样在做渲染的时候可以使用这个ICON
